static/app/components/onboarding/gettingStartedDoc/utils/feedbackOnboarding.tsx

```diff
@@ -144,11 +144,11 @@ export const getCrashReportJavaScriptInstallStep = params => [
 
 export function getCrashReportSDKInstallFirstStep(params) {
```

Comment:
is it possible to add types to the params here, and elsewhere when they are passed into & around the file? that might make it easier to spot `any` types or nullables that need special care

Reply:
this is hard because the params can be either `DocParams` (easy) or `DocParams<PlatformOptions>` and `PlatformOptions` is a specific type for each platform that has options (ex. capacitor, which has angular/vue/etc vs angular, which has angular11/angular10 etc) so i couldn't find a neat way to type this